pkgs/by-name/br/bruteforce-wallet/package.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, openssl
+, db
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bruteforce-wallet";
+  version = "1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "glv2";
+    repo = "bruteforce-wallet";
+    rev = version;
+    hash = "sha256-1sMoVlQK3ceFOHyGeXKXUD35HmMxVX8w7qefZrzAj5k=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
+
+  buildInputs = [
+    openssl
+    db
+  ];
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    description = "Try to find password of encrypted cryptocurrency wallet";
+    homepage = "https://github.com/glv2/bruteforce-wallet";
+    changelog = "https://github.com/glv2/bruteforce-wallet/blob/${src.rev}/NEWS";
+    license = licenses.gpl3Only;
```

License is [`GPL3+`](https://github.com/glv2/bruteforce-wallet/blob/master/src/bruteforce-wallet.c#L6C1-L9C38).

```suggestion
    license = licenses.gpl3Plus;
```